src/sailfishui/crash-reporter.qml

```diff
@@ -157,6 +157,32 @@ Page {
                 onClicked: PrivacySettings.allowMobileData = !PrivacySettings.allowMobileData
             }
 
+            SectionHeader {
+                //% "Battery care"
+                text: qsTrId("settings_crash-reporter_battery_care")
+            }
+
+            TextSwitch {
+                automaticCheck: false
+                checked: PrivacySettings.restrictWhenLowBattery
+                //% "Not when battery is low"
+                text: qsTrId("settings_crash-reporter_restrict_when_low_battery")
+                //% "Avoid power intensive tasks when battery is low, regardless of charger presence."
```

Comment:
Two settings for battery feel a bit overkill. A little pondering should there be even a single one and instead just do the sensible thing, i.e. avoiding heavy operations if not charging or below some battery percentage.

Reply:
If not charging or below *some* battery percentage. Imagine you have 8%, you are on a not really powerful charger, it starts uploading browser's coredumps - your device dies. Or you are below *some* %, your / is nearly full and you want to get rid of the dumps NOW. Or you don't want to be bad-surprised with battery dropped below *some* % just because the crash reporter... my *some* is 55%, yours likely different. Fortunately we have some settings available.

Reply:
8% on weak charger would skip the upload so device wouldn't die. 8% on powerful charger, yes could be case handled differently, although explicit actions could be handled specifically too. Well, I don't consider this good UI to have switch for everything, and not expecting people really bothering touching them, but minor enough detail for me not to care on a developer tool.